File: 7KYU/SImple Fun #37 House Numbers Sum/solution.cs
namespace myjinxin
{
    using System;
    using System.Linq;
    
    public class Kata
    {
        public int HouseNumbersSum(int[] inputArray)
        {
			var total = 0;
			var houses = inputArray.Reverse();
			foreach(var house in houses)
			{
				if(house == 0)
					total = 0;
				else
				total += house;
			}
			return total;
        }
    }
}

File: 7KYU/SImple Fun #37 House Numbers Sum/solution.java
public class Kata {
    public static int houseNumbersSum(final int[] arr) {
        int total = 0;
        
        for(int i = 0; i < arr.length; i++){
          if(arr[i] == 0)
            return total;
          total += arr[i];
        }
        return total;
    }
}